docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,74 @@
+
+The SQD tool can target Hamiltonians expressed as linear combinations of Pauli operators or second-quantized Fermionic operators. The input samples are obtained by quantum circuits defined by the user, which are believed to be good representations of eigenstates (for example, the ground state) of a target operator. The convergence rate of SQD as a function of the number of samples improves with the sparseness of the target eigenstate.
+
+## Install the SQD package
+
+There are two ways to install the SQD package: PyPI and building from source. It is recommended to install these packages in a [virtual environment](https://docs.python.org/3.10/tutorial/venv.html) to ensure separation between package dependencies.
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-sqd` package is via PyPI.
+
+```bash
+pip install qiskit-addon-sqd
+```
+
+### Build from source
+
+<details>
+<summary>
+Click here to read how to install this package manually
+
+</summary>
+If you wish to contribute to this package or want to install it manually, first clone the repository:
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-sqd.git
+```
+
+and install the package via `pip`. The repository also contains example notebooks that you can run. If you plan on developing in the repository, you can install the `dev` dependencies.
+
+Adjust the options to suit your needs.
+
+```bash
+pip install tox notebook -e '.[notebook-dependencies, dev]'
+```
+</details>
+
+
+## Theoretical background
+
+The SQD workflow using self-consistent configuration recovery is explained in detail in [[1]](#references). This section provides an overview to the technique depicted in the following diagram.
+
+![SQD diagram depicting configuration recovery, collecting subsamples, and obtaining eigenstates from those subsamples](/images/guides/qiskit-addons/sqd_diagram.png)
+
+Here $\bar{\mathcal{X}}$ is a set of noisy samples which contain, in the context of the Hamiltonian being simulated, physical and non-physical configurations (represented as bitstrings) obtained from execution on a QPU. The non-physical configurations are due to noise and can be processed by the `sqd.configuration_recovery.recover_configurations()` method to refine the samples into a new set $\mathcal{X}_R$. 
+
+From this set, batches of configurations $\mathcal{S}^{(1)}...\ \mathcal{S}^{(K)}$ are collected and represent samples obtained from different subspaces of the Hamiltonian. These subspace samples are built from a set of basis states, $\mathcal{S}^{(k)}=\{|\mathbf{x}^{(k)}\rangle\}$, and are projections of the system Hamiltonian, $\hat{H}$. This projection is defined as
```

Comment:
```suggestion
From this set, batches of configurations $\mathcal{S}^{(1)}...\ \mathcal{S}^{(K)}$ are collected according to a distribution proportional to the empirical frequencies of each $\mathbf{x}$  in $\mathcal{X}_R$. Each batch of sampled configurations spans a subspace, $\mathcal{S}^{(K)}$, in which the Hamiltonian is projected and diagonalized:
```

Reply:
We want to say that the Hamiltonian is projected onto the subspaces, rather than the subspaces are projections of the Hamiltonian.

I pulled this language straight from the paper, and I think it fits nicely here